tests/test_run.py

```diff
@@ -139,3 +147,312 @@ def test_train_freq_fail(train_freq):
             train_freq=train_freq,
         )
         model.learn(total_timesteps=250)
+
+
+def test_dqn_train_with_batch_norm():
```

Comment:
`test_run` are mostly for integration test (test that it runs, for the feature test, please keep it inside the `test_predict` or in another file)

Reply:
oh I see, probably this is time to create a `test_eval_mode.py`, no?

Reply:
Yes, fair point. I have moved all the tests for this to `test_train_eval_mode.py`, including the new ones that were in `test_predict.py`.

Is that ok?

Reply:
perfect =)